Dockerfile

```diff
@@ -22,14 +22,14 @@ RUN dnf update -y && \
         libnotify-devel libpeas-devel librsvg2-devel libsecret-devel \
         libzapojit-devel pango-devel polkit-devel poppler-glib-devel \
         rest-devel telepathy-glib-devel tracker-devel udisks-devel \
-        upower-devel vte{,291,3}-devel webkitgtk{,3,4}-devel
```

Comment:
No webkit docs anymore?

Reply:
Discussed (removed do to issues with compilation/breakage, will be added later)